subworkflows/local/standardisation_profiles.nf

```diff
@@ -52,12 +52,19 @@ workflow STANDARDISATION_PROFILES {
                             .map {
                                     meta, profile ->
                                         def meta_new = [:]
-                                        meta_new.id = meta.db_name
                                         meta_new.tool = meta.tool == 'malt' ? 'megan6' : meta.tool
+                                        meta_new.db_name = meta.db_name
                                         [meta_new, profile]
                             }
                             .groupTuple ()
-                            .map { [ it[0], it[1].flatten() ] }
+                            .map {
+                                meta, profiles ->
+                                    meta = meta + [
+                                        tool: meta.tool == 'kraken2-bracken' ? 'kraken2' : meta.tool, // replace to get the right output-format description
+                                        id: meta.tool == 'kraken2-bracken' ? "${meta.db_name}-bracken" : "${meta.db_name}" // append so to disambiguate when we have same databases for kraken2 step of bracken, with normal bracken
```

Comment:
More of a question but is the id referring to the sample?

Reply:
At this point no, as we want to aggregate by tool/db, as it's the taxoasta merging into one table of multiple samples

Reply:
At this point no, as we want to aggregate by tool/db, as it's the taxoasta merging into one table of multiple samples

Reply:
It is the id for distinguishing the `tool` from the `db`, then?

Reply:
yes :)

Reply:
Cool, the PR looks good to me, great job 🚀